.github/workflows/test.yml

```diff
@@ -101,6 +101,9 @@ jobs:
         if: matrix.os != 'windows-latest'
         run: make build-workspace-${{ matrix.target }}
 
+      - name: install nextest
+        run: cargo install cargo-nextest
```

Comment:
Moved here from makefile, to prevent complete rebuilds on every invocation.

Reply:
Wouldn't be a problem with `cargo-make`, could have it in both places :)